src/combination.jl

```diff
@@ -1,5 +1,6 @@
 """ This file contains all the functions related to Bliss and temporal combinations. """
 
+######---------------- Functions for Bliss combination ----------------########
 function BlissCombination(p1::Array{Float64, 2}, p2::Array{Float64, 2}, n::Int)
```

Comment:
I'd make a function that combines `BlissCombination` and `BlissModelComb`, then test that.